documentation/intermediate_ca.md

```diff
@@ -34,7 +34,7 @@ To set up Puppet as an intermediate CA with an external root:
 * The root cert
 * Any intermediate CA certs
 * The CA cert that you will use to issue a CA signing cert for your Puppet infrastructure
-2. Collect the corresponding CRLs for each of these certificates.
+2. Collect the corresponding CRLs for each of these certificates. Take note of the expiration dates of each CRL. When one expires, you need to refresh it for Puppet to continue working. For more information, see [SERVER-2535](https://tickets.puppetlabs.com/browse/SERVER-2535). 
```

Comment:
Do we often point to tickets in docs? It's not clear to me what "more information" someone is supposed to get from that ticket.

Reply:
Sometimes when it's a limitation. But you're right if there is not much more to learn from the ticket we shouldn't have it there. Thanks!